src/Log/Engine/BaseLog.php

```diff
@@ -89,20 +89,31 @@ public function scopes()
      */
     protected function _format($data, array $context = [])
     {
-        if (is_string($data)) {
+        $result = '';
+        if (isset($context['scope']) && is_array($context['scope'])) {
+            foreach ($context['scope'] as $c) :
+                $result['scope'][$c] = $data;
```

Comment:
$result is a string here. You're relying on implict array casting which looks incorrect.